runner/src/server/views/layout.html

```diff
@@ -169,21 +169,32 @@ <h1 class="govuk-heading-l">Default page template</h1>
 
 
 {% block footer %}
-    {{ govukFooter({
-        meta: {
-            items: [{
-                href: 'https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice',
-                text: 'Privacy'
-            }, {
-                href: cookiePolicyUrl,
-                text: 'Cookies'
-            }, {
-                href: accessibilityStatementUrl,
-                text: 'Accessibility Statement'
-            }, {
-                href: contactUsUrl,
-                text: 'Contact Us'
-            }]
-        }
-    }) }}
+  {% set footerItems = [] %}
+  {% if page.def.footer %}
+    {% for footerItem in page.def.footer %}
+      {% set _ = footerItems.push({
+        href: footerItem.href,
+        text: footerItem.text
+      }) %}
+    {% endfor %}  
+  {% else %}
+  {% set footerItems = [{
+        href: 'https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice',
+        text: 'Privacy'
+    }, {
+        href: cookiePolicyUrl,
+        text: 'Cookies'
+    }, {
+        href: accessibilityStatementUrl,
+        text: 'Accessibility Statement'
+    }, {
+        href: contactUsUrl,
+        text: 'Contact Us'
+    }]%}
+  {% endif %}
+  {{ govukFooter({
+    meta: {
+      items: footerItems
+    }
+  }) }}
```

Comment:
Potentially shorter version (not tested)

```suggestion
  {% set footerItems = [{
        href: 'https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice',
        text: 'Privacy'
    }, {
        href: cookiePolicyUrl,
        text: 'Cookies'
    }, {
        href: accessibilityStatementUrl,
        text: 'Accessibility Statement'
    }, {
        href: contactUsUrl,
        text: 'Contact Us'
    }]%}
  {{ govukFooter({
    meta: {
      items: page.def.footer if page.def.footer else footerItems
    }
  }) }}
```

Reply:
Actually not sure which syntax you'd need to use, but maybe could try these:

```javascript
page.def.footer || footerItems
```
```python
page.def.footer or footerItems
```

Reply:
https://mozilla.github.io/nunjucks/templating.html#if

```
{% if happy or hungry %}
  I am either happy *or* hungry; one or the other is true.
{% endif %}
```

Think this may work.  If not can just leave it